### DISCUSSION TITLE
Cord v2

### DISCUSSION THREAD

**Discussion by vyfor** [vyfor/cord.nvim#143] [Announcements]
The rewrite is complete and ready for merging. Due to the significant breaking changes and limited testing, the merge is planned for February 1, 2025, though this date may shift based on stability and other factors. Any remaining bugs are expected to be resolved by then. We encourage you to migrate to Cord v2 and assist us in hunting down any bugs or issues before the merge. Feedback is highly valued and greatly appreciated.

- [Cord v2](https://github.com/vyfor/cord.nvim/blob/client-server)
- Tracking Issue: #133
- PR: #131

UPDATE: Merged! Huge thanks to the beta testers! 🙌

**Comment by alsaiduq-lab** [comment]
Been working well on my end. I really like the new changes.

**Comment by Eveeifyeve** [comment]
I have a question, why is the icons now located in a separate repo?

**Reply by vyfor** [reply]
https://github.com/vyfor/cord.nvim/issues/116
> As the project grows, there will be a need to populate the assets with new icons. Since icons are pulled from GitHub anyway, keeping them in the current repository only adds to the data you have to pull when installing the plugin. We will likely move the icons to their own repo.

**Reply by Eveeifyeve** [reply]
Have you considered making a GitHub org for cord-nvim? As you could store cord-nvim there as well as icons and maybe even a potential website for the documentation like an mdbook for example?

**Reply by vyfor** [reply]
Yes, but I don't think it's necessary since it's not likely to expand beyond just the icons and the plugin itself. I might make use of the icons for other projects as well. I may reconsider later, but not at the moment.
